config/initializers/1_redirect_stdout.rb

```diff
@@ -0,0 +1,6 @@
+# In Passenger environment standard output and error is not logged at all.
+# Using this technique it is possible to redirect all the output to standard
+# Rails log.
+
+$stderr.reopen Rails.logger.instance_variable_get(:@logdev).dev
+$stdout.reopen Rails.logger.instance_variable_get(:@logdev).dev
```

Comment:
Ok added this. Might be dangerous if someone closes (or reopens) the device. Alternative might be an IO wrapper: http://stackoverflow.com/questions/9637092/redirect-stderr-to-logger-instance